packages/providers/providers.yaml

```diff
@@ -806,6 +806,24 @@ asana:
             limit_name_in_request: limit
     docs: https://docs.nango.dev/integrations/all/asana
 
+asana-mcp:
+    display_name: Asana (MCP)
+    categories:
+        - productivity
+        - ticketing
+        - mcp
+    auth_mode: MCP
```

Comment:
I don't think MCP as `auth_mode` is correct, currently it's oauth providers but what if they add support for basic auth then you are limited. 
At least putting MCP_OAUTH2 would be better

Reply:
Or the opposite. MCP is the default which uses OAUTH and if they add in basic then it becomes MCP_BASIC

Reply:
IMO MCP_OAUTH2 makes sense as it's explicit and we might see a bunch of MCP auth modes.

Reply:
Updated with https://github.com/NangoHQ/nango/pull/4660/commits/63633d2c0b1e2c2405a7d9092bc9ea6bc37e2181